kb-exchangerates

```diff
@@ -0,0 +1,24 @@
+# International Payment, Currency and Exchange
```

Comment:
I don't see any of the metadata required on any kb article.
refer to https://github.com/CenturyLinkCloud/PublicKB/blob/master/README.md